subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/maven/MavenGoogleDependencyResolveIntegrationTest.groovy

```diff
@@ -24,7 +24,11 @@ import org.gradle.util.TestPrecondition
 class MavenGoogleDependencyResolveIntegrationTest extends AbstractDependencyResolutionTest {
 
     def setup() {
-        buildFile << googleRepository()
+        buildFile << """
+            repositories {
```

Comment:
This was changed because the test actually cares about validating our integration with the real google repository. We shouldn't use our mirror in that case.